src/utils/validators.js

```diff
@@ -28,6 +28,7 @@ export const TIKTOK_REGEX = ".com/@)([a-zA-Z0-9_-]+([/.])?)+$"
 // 4. The "^" and "$" ensures that we are matching the whole string (to prevent the user from adding "https://")
 export const DOMAIN_NAME_REGEX =
   "^([a-z0-9]([a-z0-9-]{0,61}[a-z0-9])?\\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]$"
+export const INSTAGRAM_POST_URL_REGEX = `${URL_REGEX_PREFIX}instagram.com/p/)[a-zA-Z0-9_-]+/?`
```

Comment:
this regex might be wrong - there's a unmatched `)` (after `instagram.com/p/`) and the `/?` seems to suffer from the same problem as the one [here](https://github.com/isomerpages/isomercms-frontend/pull/1045)

Reply:
The unmatched `)` comes from the `URL_REGEX_PREFIX`, as there is an opening `(` from there which will need to be matched in this string.

Reply:
^ oof didn't realise this ;--; could we add a comment wrt this terminating `)`